### ISSUE TITLE
grid shrink properties? [8:33am]

### ISSUE THREAD

**Issue by frankcollins3** [frankcollins3/Next-Water-App#131]
`attempting to do:`
redesign -> redeploy. 
![Screen Shot 2024-06-17 at 8 32 51 AM](https://github.com/frankcollins3/Next-Water-App/assets/73137934/f383edf7-4948-4a13-a44b-14b74ebe21a3)

// keep in mind: intro page & part of redesign-approach allows user to see a few in-app-things they couldn't before (w.o login)

`error:`
// this also goes for prospective interviews (granted there are pictures and videos around but still)
in my opinion the interviewer should not have to signup to see most of the design (which they likely never have time for) 

the original app doesn't allow user to see the schedule or the settings without creating an account which is sensible software.

I didn't even design the original `<settings>` or schedule so I'm not complimenting myself when I say I think it looks great.

`proposed approach:`

**Comment by frankcollins3** [comment]
TL:DR; 
if we are limited to mere seconds of eye-time from an interviewer / prospective employer 
then why hide most impactful design features behind loginwall which the interviewer won't have time to overcome.  

dissection debuts:

// more opinions:
we spend enough time coding in the scape to come to fact: don't need a drastic change every time you hover over something

similarly, after certain point, hiding settings from user would would seem same. 
It throws away the interviewer missing impact of everything they'd see since they won't have time to signup cant login cant see features.
It's like a "hill willing to die on" 
However much impact you lose esp for FE web-dev roles, you'd completely gain it all back but would have to allow users to see those features.. 
If it's not a startup then why hide them on a personal project. Almost seems like hubris thinking they'll spent 3 minutes on it
I've had summer job and been final-4-of-shortlist and one could think the interviewer would be insane then not to spend 3-5 minutes on the last 3-5 people but that's a huge gamble.
